packages/core/src/odata-common/selectable/orderable-edm-field.ts

```diff
@@ -0,0 +1,58 @@
+import { Filter } from '../filter';
+import { Entity } from '../entity';
+import { EdmTypeShared } from '../edm-types';
+import { EdmField, FieldTypeByEdmType } from './edm-field';
+
+export class OrderableEdmField<
```

Comment:
Also ordreable is a boolean like property. You could either pass the `NonOderType` or the `OrderType` instead of true/false where the `OrderType` class implements the methods. Doing so you could even break the inheritance chain in the fields.

Reply:
I have thought about this 3mins so this is just a basis for discussion.

Reply:
I was also considering `Sortable` as a keyword or `GreaterOrLess`. I like none of them. `Orderable` could easily be confused with the actual `Orderable` interface (asc, desc), but this keyword is already used in another place.

Reply:
Won't fix, as discussed.